src/WindowManager/WindowManager.ts

```diff
@@ -20,6 +20,8 @@ export class WindowManager {
   private _walletUrl = '';
   protected initialized = false;
   public walletWindow: Window | null = null;
+  private activeListeners: Map<string, (event: MessageEvent) => void> =
+    new Map();
```

Comment:
can we use an object instead of Map and Object.keys ?

Reply:
in this case, for performance it is better to use Map instead of objects